sdk/src/main/java/com/horizen/account/api/rpc/utils/RpcCode.java

```diff
@@ -0,0 +1,35 @@
+package com.horizen.account.api.rpc.utils;
+
+public enum RpcCode {
+    // default json-rpc error codes
+    ParseError(-32700, "Parse error"),
+    InvalidRequest(-32600, "Invalid request"),
+    MethodNotFound(-32601, "Method not found"),
+    InvalidParams(-32602, "Invalid params"),
+    InternalError(-32603, "Internal error"),
+
+    // custom ethereum error codes
+    Unauthorized(1, "Unauthorized"),
+    ActionNotAllowed(2, "Action not allowed"),
+    ExecutionError(3, "Execution error"),
+    ExecutionReverted(-32000, "Execution reverted"),
+    UnknownBlock(-39001, "Unknown block");
```

Comment:
don't see this error code in geth. Need to check.
In general, are we sure that all the codes are found and properly used by us?